### ISSUE TITLE
Testng feature

### ISSUE THREAD

**Issue by RafalJDev** [krzysieksr/testListsA#3]
Jeśli używasz testng to masz parametr w adnotacji dla wyjątków @Test, np. @Test(expectedExceptions = NullPointerException.class)

**Comment by krzysieksr** [comment]
Tak, ale zanim zacząłem używać TestNG testowałem z try..catch, wiec postanowiłem trzymać się tego już do konca:)
